flexget/utils/bittorrent.py

```diff
@@ -252,6 +252,11 @@ def get_filelist(self):
         return files
 
     @property
+    def name(self):
+        """Return name of the torrent"""
+        return self.content['info'].get('name', '')
```

Comment:
My point was rather, why return something specific at all? `get` will return `None` if the key isn't set.